docs/guides/postinstall.md

```diff
@@ -172,9 +174,44 @@ S3 suspend has been broken since macOS Sonoma, it has never been fixed, but this
      [Install]
      WantedBy=sleep.target
      ```
+3. Check your modprobe location
+     ```bash
+     which modprobe
+     which rmmod
+     ```
+and fix the pathes in service script if they differ for your system. If you having problems with touchbar being dead after restoring state from suspend then uncomment ExecStartPre line.
 
-3. Enable the service by running: `sudo systemctl enable --now suspend-fix-t2.service`
+5. Enable the service by running: `sudo systemctl enable --now suspend-fix-t2.service`
 
 !!! note
-    This seems to be working only on Arch with `CONFIG_MODULE_FORCE_UNLOAD=y` in the kernel config.
-    To check, run: `zcat /proc/config.gz | grep "CONFIG_MODULE_FORCE_UNLOAD"`
+
+    This seems to be working with `CONFIG_MODULE_FORCE_UNLOAD=y` in the kernel config when kernel is compiled with [kernel compilation instructions](https://wiki.t2linux.org/guides/kernel/).
```

Comment:
Why specifically kernel compilation instructions? There are people who know how to compile a kernel better than this guide.